src/routes/(site)/blog/[id]/+page.svelte

```diff
@@ -5,8 +5,24 @@
   import { Temporal } from '@js-temporal/polyfill';
   import Labels from '$lib/components/blog/labels.svelte';
   import BlogBody from './blog-body.svelte';
-
+  import { onMount } from 'svelte';
+  import { toast, ToastType } from '$lib/components/toaster/toasts';
   export let data: PageData;
+  onMount(() => {
+    const copyBtns = document.querySelectorAll('.copy-code');
+    for (let i = 0; i < copyBtns.length; i++) {
+      copyBtns[i].addEventListener('click', (event: any) => {
```

Comment:
remove any, use proper type for this

event.target is type "EventTarget" and not have a .offsetParent attribute
<img width="402" alt="image" src="https://user-images.githubusercontent.com/8981287/196590027-932ccab1-3e36-4e68-9a91-6ad17a52c2d5.png">

result in runtime error when clicked on vercel

<img width="933" alt="image" src="https://user-images.githubusercontent.com/8981287/196589907-f1d5e54a-0bbe-45da-872f-4134d4d1c45a.png">

Reply:
Update: works on local

Reply:
```ts
        const content =
          (event.target as HTMLElement).offsetParent?.attributes.getNamedItem(
            'data-snippet-clipboard-copy-content'
          )?.value ?? '';
```

Reply:
done